crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1390,11 +1390,19 @@ impl World {
         self.change_tick.fetch_add(1, Ordering::AcqRel)
     }
 
+    /// Reads the current change tick of this world.
```

Comment:
Super basic question so I'm sorry lol, but is a change tick different than a normal tick? 

I understand a `tick` to be a "loop" of the main update loop. But in ECS's I suppose it might not be that simple. I guess this tick is specific to the world?

Reply:
I suppose maybe I need some place to read up about how `ticks` are done in ECS's since maybe worlds and systems run at different ticks? Is there a good place to read up on that?

Reply:
Basically, a change tick is a number signifying when a system ran in relation to other systems. This is used to answer questions like "did this value change since the last time this system ran?", powering change detection.

I don't know of any write-ups on change detection internals, so I'd recommend reading this source files for more details:

https://github.com/bevyengine/bevy/blob/main/crates/bevy_ecs/src/change_detection.rs
https://github.com/bevyengine/bevy/blob/main/crates/bevy_ecs/src/component.rs#L521

I suppose the docs could use some work, here.

Reply:
thank you so much!